catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorColorVisualizeTest.kt

```diff
@@ -0,0 +1,219 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2023 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.formulaeditor
+
+import android.text.Spannable
+import android.view.View
+import android.widget.TextView
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import org.catrobat.catroid.ProjectManager
+import org.catrobat.catroid.R
+import org.catrobat.catroid.content.bricks.SetColorBrick
+import org.catrobat.catroid.formulaeditor.VisualizeColorImageSpan
+import org.catrobat.catroid.ui.SpriteActivity
+import org.catrobat.catroid.uiespresso.content.brick.utils.ColorPickerInteractionWrapper
+import org.catrobat.catroid.uiespresso.formulaeditor.utils.FormulaEditorWrapper.Category.LOGIC
+import org.catrobat.catroid.uiespresso.formulaeditor.utils.FormulaEditorWrapper.FORMULA_EDITOR_TEXT_FIELD_MATCHER
+import org.catrobat.catroid.uiespresso.formulaeditor.utils.FormulaEditorWrapper.onFormulaEditor
+import org.catrobat.catroid.uiespresso.util.UiTestUtils.Companion.createProjectAndGetStartScript
+import org.catrobat.catroid.uiespresso.util.rules.FragmentActivityTestRule
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.koin.java.KoinJavaComponent
+
+class FormulaEditorColorVisualizeTest {
+
+    private val COLOR_STRING_CONVERSION_CONSTANT = 16
+
+    val projectManager: ProjectManager by KoinJavaComponent.inject(ProjectManager::class.java)
```

Comment:
import org.koin.java.KoinJavaComponent.inject and then just use inject(ProjectManager::class.java)